resources/views/common/navbar.blade.php

```diff
@@ -0,0 +1,30 @@
+<header id="header">
+  
+  <div class="nav">
+    <input type="checkbox" id="nav-check">
+    <div class="nav-header">
+      <div class="nav-title">
+        INTERCCON
+      </div>
+    </div>
+    <div class="nav-btn">
+      <label for="nav-check">
+        <span></span>
+        <span></span>
+        <span></span>
+      </label>
+    </div>
+    
+    <div class="nav-links">
+      <a href="#about">About</a> 
+      <a href="#services">Services</a> 
+      <a href="/register">Register</a> 
+      <a href="/login">Login</a> 
+      <a href="#business">Make Order</a> 
```

Comment:
I am looking at a scenario when I am logged in. Why would I still need to see the register and loin buttons?

Reply:
@reifred other fields kindly detail and suggest any you have.

Reply:
It's good you discuss with the team what would be best. But for my suggestions, I detailed everything, you just didn't consider it. 
![image](https://user-images.githubusercontent.com/22131999/70181529-1f869880-16f3-11ea-8613-f32b4d112bc4.png)

Reply:
I have fixed this as well boss.

Reply:
Thanks for your persistence and the good work on the navigation. The fix is good on large devices but when I tested it on small devices. The navigation menu can't be seen, even after pressing the navbar toggle.
![image](https://user-images.githubusercontent.com/22131999/70803177-81ff2900-1dc4-11ea-9b8d-b946df62a048.png)